frontend/src/components/Chat/DirectMessageSelect.tsx

```diff
@@ -36,8 +45,8 @@ export default function DirectMessageSelect({
     onDirectChatOpen(userName);
   };
 
-  const renderNotification = (player: Player): JSX.Element | null => {
-    const directMessageId = [player.id, myPlayerID].sort().join(':');
+  const renderNotification = (participant: DirectMessageParticipant): JSX.Element | null => {
+    const directMessageId = [participant.userId, myPlayerID].sort().join(':');

```

Comment:
It doesn't look like you're actually using this anywhere?

Reply:
it gets called on line 87